tensor2tensor/data_generators/text_encoder.py

```diff
@@ -36,6 +37,10 @@
 EOS = '<EOS>'
 RESERVED_TOKENS = [PAD, EOS]
 
+# Character that will be used instead of those with
+# ordinal numbers > 255
+UNK_CHAR = '×' # ordinal 215
```

Comment:
Because it looks kinda like an "unknown" token and is unlikely to clash with significant characters. We have to do something with Unicode characters with ordinals > 255, and mapping them to an 'indicator' character instead of skipping them feels sensible.